src/snowflake/snowpark/session.py

```diff
@@ -2417,7 +2417,16 @@ def _infer_is_return_table(
         self, sproc_name: str, *args: Any, log_on_exception: bool = False
     ) -> bool:
         func_signature = ""
+        sproc_name = sproc_name.strip().upper()
         try:
+            # do not run describe procedure on system procedures
+            if sproc_name in [
+                "ASSOCIATE_SEMANTIC_CATEGORY_TAGS",
```

Comment:
Are these the only system functions we care about? If this list is going to potentially grow or used elsewhere does it make sense to create a static list variable for it?

Reply:
For now I don't foresee it to be used elsewhere but system defined stored procs that are available to users may grow.

Reply:
![Screenshot 2023-12-15 at 11 40 00 AM](https://github.com/snowflakedb/snowpark-python/assets/107440467/6832edbf-182b-4a75-ac0d-ab07c13794f7)
For now the `created_on` field for all these system functions is `2012-08-01 00:00:00.000 -0700`